engine-rest/engine-rest-openapi/src/main/templates/paths/history/process-instance/post.ftl

```diff
@@ -78,7 +78,8 @@
                                     "superCaseInstanceId":null,
                                     "caseInstanceId":null,
                                     "tenantId":null,
-                                    "state":"ACTIVE"
+                                    "state":"ACTIVE",
```

Comment:
Validated in swagger editor

<details>
<summary>Screenshot</summary>
<img width="1244" alt="Screenshot 2024-09-09 at 4 25 57 PM" src="https://github.com/user-attachments/assets/2af758b6-6108-4ddb-9c64-0e96c5c71bf2">
</details>